low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -23,15 +23,18 @@ import sys
 import shutil
 import json
 import subprocess
+import glob
 
 from framework.base.sspl_constants import (PRODUCT_FAMILY, SUPPORT_REQUESTOR_NAME,
     SUPPORT_EMAIL_ID, SUPPORT_CONTACT_NUMBER, ENCL_TRIGGER_LOG_MAX_RETRY,
-    ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, CONSUL_DUMP,
-    DATA_PATH)
+    ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, DATA_PATH)
 from framework.utils.conf_utils import Conf, SSPL_CONF
 
-onGoingRequestPath = f"{DATA_PATH}sspl-sb-req.txt"
-sspl_bundle_tmp = f"{DATA_PATH}sspl_bundle_{str(int(time.time()))}"
+SB_DATA_PATH = DATA_PATH + 'support_bundle'
```

Comment:
why cant we maintain these variables as class vars only? If we happen to expose interface for this script also, arranging in class vars & members would be useful in that case

Reply:
We can't make this as class vars because we are also using that vars out of the class.
Please check line no. 276

Reply:
Ok this is that marker file, stating ongoing support bundle request. And if we keep this reference check outside class, like its there right now, anyone importing this class will be able to execute support bundle without this already running support-bundle check. So its even more imp to have it in class i feel, initialize can check & error out then if file exists

Reply:
Done